packages/voyager-keycloak/src/KeycloakSecurityService.ts

```diff
@@ -73,4 +74,41 @@ export class KeycloakSecurityService implements SecurityService {
       res.json({})
     })
   }
+
+  public async validateToken(token: string): Promise<boolean> {
+    const tokenObject = this.getTokenObject(token)
+    const result = await this.keycloak.grantManager.validateToken(tokenObject, 'Bearer')
+    return (result === tokenObject) ? true : false
+  }
+
+  public getTokenObject(token: string): KeycloakToken {
+    let fullToken: KeycloakToken = {}
+    if (token) {
+      try {
+        const parts = token.split('.')
```

Comment:
If this is copy paste I will put that somewhere outside this file and link to original code 
so we could have simple way to update it in case of potential problems etc.